doc/tutorial.txt

```diff
@@ -40,33 +40,31 @@ For rt-app:
 
 export ac_cv_lib_json_c_json_object_from_file=yes
 ./autogen.sh
-./configure --host=aarch64-linux-gnu LDFLAGS=" --static -L<path to parent of json repo>/json-c/." CFLAGS="-I<path to parent of json repo>" --with-deadline
-make
+./configure --host=aarch64-linux-gnu LDFLAGS="-L<absolute path to json repo>/json-c" CFLAGS="-I<path to json repo>" --with-deadline
+AM_LDFLAGS="-all-static" make
 
 e.g, with a directory structure like the following:
 
-$ tree -d
-.
-├── autom4te.cache
-├── build
-├── build-aux
-├── doc
-│   └── examples
-│       ├── cpufreq_governor_efficiency
-│       ├── merge
-│       └── tutorial
-├── json-c
-│   ├── autom4te.cache
-│   ├── doc
-│   │   └── html
-│   └── tests
-├── libdl
-├── m4
-└── src
+$ tree -d -L 2
+|-- json-c
+|   |-- autoconf-archive
+|   |-- autom4te.cache
+|   |-- cmake
+|   |-- fuzz
+|   `-- tests
+`-- rt-app
+    |-- autom4te.cache
+    |-- build-aux
+    |-- doc
+    |-- libdl
+    |-- m4
+    `-- src
+
 
 the configure step becomes
 
-./configure --host=aarch64-linux-gnu LDFLAGS="--static -L./../json-c/." CFLAGS="-I./.." --with-deadline
+cd rt-app
+./configure --host=aarch64-linux-gnu LDFLAGS="-L$PWD/../json-c" CFLAGS="-I./../json-c" --with-deadline
```

Comment:
Sorry for getting back again on this, but with a tree like the above this triggers the following
```
rt-app_parse_config.c:28:10: fatal error: json-c/json.h: No such file or directory
 #include <json-c/json.h>
          ^~~~~~~~~~~~~~~
compilation terminated.
```
Changing the configure step to
```
./configure --host=aarch64-linux-gnu LDFLAGS="-L$PWD/../json-c" CFLAGS="-I./../.." --with-deadline
```
Does the trick, so it looks like it's actually
```
./configure --host=aarch64-linux-gnu LDFLAGS="-L<absolute path to json repo>/json-c" CFLAGS="-I<path to parent of json repo>" --with-deadline
```

Reply:
Yep this works and it seems the best solution to me as well.

Reply:
i tested it on my machine but i had the headers installed in /usr/include.. sorry.